.github/workflows/dev.yaml

```diff
@@ -0,0 +1,154 @@
+name: Release to dev
+on:
+  push:
+    branches:
+      - dev-cere
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: [self-hosted, cere-network-large-workers]
+    outputs:
+      version: ${{ steps.version.outputs.version }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Process version
+        id: version
+        run: |
+          echo "::set-output name=version::dev-${{ github.run_attempt}}-$(git rev-parse --short HEAD)"
+      - name: Configure AWS credentials for push images
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.ORG_AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.ORG_AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Build and push docker image for cerestats-api
+        env:
+          ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+          ECR_REPOSITORY: crb-cerestats-api
+        uses: docker/build-push-action@v2
+        with:
+          context: ./
+          file: ./api/Dockerfile
+          push: true
+          no-cache: true
+          build-args: |
+            NPM_TOKEN=${{ secrets.NPM_TOKEN }}
+          tags: ${{ env.ECR_REGISTRY }}/${{ env.ECR_REPOSITORY }}:${{ steps.version.outputs.version }}
+      - name: Build and push docker image for cerestats-crawler
+        env:
+          ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+          ECR_REPOSITORY: crb-cerestats-crawler
+        uses: docker/build-push-action@v2
+        with:
+          context: ./
+          file: ./backend/docker/crawler/Dockerfile
+          push: true
+          no-cache: true
+          build-args: |
+            NPM_TOKEN=${{ secrets.NPM_TOKEN }}
+          tags: ${{ env.ECR_REGISTRY }}/${{ env.ECR_REPOSITORY }}:${{ steps.version.outputs.version }}
+
+  build_cerestats_frontend:
+    runs-on: [ self-hosted, cere-network-large-workers ]
+    container:
+      image: node:14-buster-slim
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install system packages
+        run: |
+          apt update
+          apt install -y python3 libpq-dev make build-essential
+      - name: Restore cache
+        uses: actions/cache@v2
+        with:
+          path: 'node_modules'
+          key: ${{ runner.os }}-modules-${{ hashFiles('yarn.lock') }}
+      - name: Install packages
+        run: |
+          cp -rf frontend/frontend.config-cere-dev.js frontend/frontend.config.js
+          yarn install
+      - name: Build
+        run: yarn workspace frontend generate
+      - name: Prepare artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: dist
+          path: frontend/dist
+
+  deploy:
+    runs-on: [self-hosted, cere-network-dev-deployer]
```

Comment:
Let's follow the same format everywhere. In line 58 you have spaces after and before square brackets.